setup/inc/class.installer.php

```diff
@@ -318,6 +318,8 @@ function install($vars) {
             $user->setOrganization($org);
         }
 
+        // Flush the object cache
+        ModelInstanceManager::flushCache();
```

Comment:
Perhaps this needs to be in `DynamicForm::rebuildDynamicDataViews();` function? That way if it's called elsewhere it does the cache flush?